prov/net/src/xnet_eq.c

```diff
@@ -118,6 +124,52 @@ static struct fi_ops xnet_eq_fi_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static int xnet_eq_wait_try_func(void *arg)
+{
+	(void)arg;
```

Comment:
We usually use OFI_UNUSED() macro instead, as the definition changes per OS.

Reply:
Done